docs/pre-release/reference/vpn.md

```diff
@@ -0,0 +1,157 @@
+<!-- TODO: We should figure out a way to make this site-wide without affecting the release notes page -->
+<div class="container">
+
+# Telepresence and VPNs
+
+## The test-vpn command
+
+You can make use of the `telepresence test-vpn` command to diagnose issues
+with your VPN setup.
+It will guide you through a series of steps to figure out if there are
+conflicts between your VPN configuration and telepresence.
+
+### Prerequisites
+
+Before running `telepresence test-vpn` you should ensure that your VPN is
+in split-tunnel mode.
+This means that only traffic that _must_ pass through the VPN is directed
+through it; otherwise, the test results may be inaccurate.
+
+You may need to configure this on both the client and server sides.
+Client-side, taking the Tunnelblick client as an example, you must ensure that
+the `Route all IPv4 traffic through the VPN` tickbox is not enabled:
+
+<img src="../../images/tunnelblick.png" />
+
+Server-side, taking AWS' ClientVPN as an example, you simply have to enable
+split-tunnel mode:
```

Comment:
I think the idea is to show users what the settings may look like. There are tons of VPNs and we want to show them what to look for with an example (I don't think we could realistically create a guide for each VPN).  

I think the example is helpful since a lot of people probably aren't familiar with these settings or terminology (I know I wasn't) so seeing how you do it in a few different examples would be helpful? Maybe there's a way we can better word that this is what we are trying to do?

Reply:
yeah, like Donny said i think a lot of users aren't even aware that these options are available in their VPN providers/clients, so i kinda wanted to show one full example to make it all a little more concrete